docs/PFCWD-test-plan.md

```diff
@@ -0,0 +1,189 @@
+
+•	Test specific configurations are mentioned in respective test cases.
+
+## 1.3 Test Cases
+
+### 1.3.1 Test Case #1 - PFCWD two senders two receivers
+
+#### 1.3.1.1 Test Objective
+
+This test aims to verify how PFC watchdog can handle PFC storms in a topology with two senders and two receivers.
+
+#### 1.3.1.2 Test Topology
+
+Refer to Topology 1.1.2.1 for the test topology.
+
+#### 1.3.1.3 Test Configuration
+
+- On SONiC DUT configure the following:
+  1. Enable watchdog with default storm detection time (400ms) and restoration time (2sec). Please note that different switches have different detection times and restoration times, e.g., 200ms and 400ms.  
+  2. Configure a single lossless priority value Pi (0 <= i <= 7).
+  3. To minimize configuration complexity, it is recommended that the SONiC DUT be configured as either Top of Rack (ToR) / Tier 0 (T0) switch with three VLAN interfaces.
+
+- Configure following traffic items on the Keysight device:
+  1. Traffic 1<->2 : Bi-directional traffic between Keysight port 1 and port 2, with DSCP value mapped to lossless priority Pi configured in the DUT. Traffic Tx rate should be configured as 50% of line rate.
+  2. Traffic 2<->3 : Bi-directional traffic between Keysight port 2 and port 3, with DSCP value mapped to lossless priority Pi configured in the DUT. Traffic Tx rate should be configured as 50% of line rate.
+  3. PFC pause storm: Persistent PFC pause frames from Keysight
+        port 3 to et3 of DUT. Priority of the PFC pause
+        frames should be same as that configured in DUT and the
+        inter-frame transmission interval should be lesser than
+        per-frame pause duration.
+
+#### 1.3.1.4 Test Steps
+
+Refer to the time diagram below to understand the work flow of the test case:
+
+![](image/PFCWD_2rcvrs_2senders_test_workflow.PNG)
+
+*Fig. 2: #Test 1 Workflow*
+
+1. At time **_T<sub>startTraffic</sub>_** , start all the bi-directional lossless traffic items.
+2. At time **_T<sub>startPause</sub>_** , start PFC pause storm.
+3. At time **_T<sub>stopPause</sub>_** , stop PFC pause storm. (**_T<sub>stopPause</sub> - T<sub>startPause</sub>_**) should be larger than PFC storm detection time to trigger PFC watchdog.
+4. At time **_T<sub>stopTraffic</sub>_** , stop lossless traffic items. Note that (**_T<sub>stopTraffic</sub> - T<sub>stopPause</sub>_**) should be larger than PFC storm restoration time to re-enable PFC.
+5. Verify the following:
+   * PFC watchdog is triggered on the corresponding lossless priorities at DUT interface et3. 
+   * 'Traffic 1<->2' must not experience any packet loss in both directions. Its throughput should be close to 50% of the line rate.
+   * For 'Traffic 2<->3', between **_T<sub>startPause</sub>_** and **_T<sub>stopPause</sub>_** , there should be some packet loss in both directions. 
+   * After **_T<sub>stopPause</sub>_** , the traffic throughput should gradually increase and become 50% of line rate in both directions. 
```

Comment:
This is totally wrong. You mix two test cases

Reply:
Modification as per suggestion in meeting. We will add new test which you will be sending out in new revision of test plan.